src/app/shared/models/fcmNotificationDto.ts

```diff
@@ -0,0 +1,12 @@
+export interface FcmNotificationDto {
```

Comment:
I think all of these interfaces can be put together in a single file.

Reply:
and I would remove the Dto part here. If you think it is confusing, we can keep it.